tests/functional/demo/test_demo.py

```diff
@@ -538,3 +538,26 @@ def test_demo_with_nondefault_config_keyword(database, monkeypatch):
     assert (params[0]['value'] - 34.56789) < 1e-5
 
     orion.core.config.user_script_config = 'config'
+
+
+@pytest.mark.usefixtures("clean_db")
+@pytest.mark.usefixtures("null_db_instances")
+def test_demo_precision(database, monkeypatch):
+    """Test a simple usage scenario."""
+    monkeypatch.chdir(os.path.dirname(os.path.abspath(__file__)))
+
+    user_args = [
+        "-x~uniform(-50, 50, precision=5)"]
+
+    orion.core.cli.main([
+        "hunt", "--config", "./orion_config.yaml", "./black_box.py"] + user_args)
```

Comment:
This should solve the issue.
```suggestion
        "hunt", "--config", "./orion_config.yaml", "--max-trials", "2", "./black_box.py"] + user_args)
```